components/style/gecko/selector_parser.rs

```diff
@@ -264,6 +274,24 @@ impl<'a> ::selectors::Parser for SelectorParser<'a> {
         }
     }
 
+    fn parse_non_ts_functional_pseudo_class(&self, name: Cow<str>, parser: &mut CssParser)
+                                            -> Result<NonTSPseudoClass, ()> {
+        match_ignore_ascii_case! { &name,
+            "-moz-any" => {
+                // https://developer.mozilla.org/en-US/docs/Web/CSS/:any
+                // The selectors may not contain pseudo-elements and the only combinator allowed is
```

Comment:
The devmo docs don't seem to match the code.  The code has:


    // Check that none of the selectors in the list have combinators or
    // pseudo-elements. 

and proceeds to do just that.  See http://searchfox.org/mozilla-central/rev/ca7015fa45b30b29176fbaa70ba0a36fe9263c38/layout/style/nsCSSParser.cpp#6511-6518